src/test/scala/mesosphere/marathon/api/akkahttp/v2/GroupsControllerTest.scala

```diff
@@ -125,35 +116,30 @@ class GroupsControllerTest extends UnitTest with ScalatestRouteTest with Inside
       }
     }
 
-    "show app in a given version for root group" in {
-      val infoService = mock[GroupInfoService]
-      infoService.selectGroupVersion(eq(PathId.empty), any, any, any) returns Future.successful(Some(GroupInfo(createGroup(PathId.empty), None, None, None)))
+    "show app in a given version for root group" in new Fixture {
```

Comment:
Hm, I don't think `new Fixture` is required here since a fixture is created below.